src/WS2812FX/effects/BeatsinGlowEffect.cpp

```diff
@@ -4,9 +4,10 @@
 
 bool BeatsinGlowEffect::init(WS2812FX* strip) {
     // Use standard initialization pattern from helper
-    bool initialized = false;
+    bool tempInit = false;
```

Comment:
seems to only feed the standardInit function of "Effecthelper".

Actually, it would be wise to move the standardInit function to the base (template) Effect Class. This way it has access to the internal _isInitialized variable or the respective getter / setter and it could be used in the beginning of  the init() function for each class.

Alternatively, one could have a default init() function which is only overwritten if a customized init function is neccessary for the effect.

Please implement the more efficient variant.